atm.js

```diff
@@ -1,9 +1,13 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checking_balance = document.querySelector("#checking_balance");
+checkingBalance = parseInt( checking_balance.innerHTML.replace("$", "") );
+savings_amount = querySelectorall("savings_deposit").value;
+checking_amount = querySelectorall("savings_deposit").value;
```

Comment:
This will cause `Uncaught ReferenceError: querySelectorall is not defined`. use `document.querySelectorAll()` or `$("savings_deposit")` note: `.` in selector string